README.md

```diff
@@ -22,15 +20,15 @@ What i did  in this project:
 
 ## Live Demo
 
-[Live Demo Link](https://github.com/dami1080/my-portfolio/tree/contact-form)
+[Live Demo Link](https://github.com/dami1080/)
```

Comment:
- [ ] Please provide a valid live demo link or just delete its section since it's not a hard requirement for this project.
Kindly remember that such a small mistake might leave a bad impression on future viewers.

- [ ] Also, kindly fix the MIT broken link for the same reason.